src/components/NavigationPanel.js

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import LogoutButton from './LogoutButton';
 
 const NavigationPanel = () => (
   <nav>
     <Link to="/homepage">Doctors</Link>
+    <br />
+
+    {localStorage.getItem('userData') && (
+    <>
+      <Link to="/add-item">Add Item</Link>
+    </>
```

Comment:
With regards to the project requirements, Please, kindly update your routes to 'Reserve form'

Reply:
Well noted @oguntayomicheal changes made and final merged.Thanks